macbuild.tool

```diff
@@ -6,7 +6,8 @@ package() {
     exit 1
   fi
 
-  local ver=$(cat Include/OpenCore.h | grep OPEN_CORE_VERSION | sed 's/.*"\(.*\)".*/\1/' | grep -E '^[0-9.]+$')
+  local ver
+  ver=$(cat Include/OpenCore.h | grep OPEN_CORE_VERSION | sed 's/.*"\(.*\)".*/\1/' | grep -E '^[0-9.]+$')
```

Comment:
Why is that needed?

Reply:
macbuild.tool works perfectly well as it is ... run it dozens of times in last week.

So change not "Needed" ... just desirable in terms of maintaining standards.
I.E., no issue here but there is a potential issue that may happen elsewhere in development if such local variables are declared and assigned together.

See: https://github.com/koalaman/shellcheck/wiki/SC2155#problematic-code-in-the-case-of-local